res/themes/dark/css/_dark.scss

```diff
@@ -272,6 +272,10 @@ $composer-shadow-color: rgba(0, 0, 0, 0.28);
             background-color: #080808;
         }
     }
+
+    blockquote {
+        
+    }
```

Comment:
shouldn't there be something in here?
I'm surprised the linter hasn't harassed you about an empty block

Reply:
Gottem 😎